core/domain/classifier_services.py

```diff
@@ -444,6 +444,8 @@ def load_examples(self, examples):
 
     def predict_label_for_doc(self, d):
         """Returns the predicted label from a doc's prediction report."""
+        if self._num_docs < 20 or self._num_labels < 3:
```

Comment:
Yes very good point. Sean and I discussed this but I totally forgot to describe here. Will add some docs.